cdap-watchdog/src/main/java/co/cask/cdap/metrics/store/DefaultMetricStore.java

```diff
@@ -113,6 +113,11 @@ public Cube get() {
       // i.e. for flows only
       ImmutableList.of(Constants.Metrics.Tag.NAMESPACE, Constants.Metrics.Tag.APP,
                        Constants.Metrics.Tag.FLOW)));
+    // queue
+    aggs.put("queue", new DefaultAggregation(
```

Comment:
I don't really need instance id and run id, at least not to show total pending for the queue in the flow status. But an operator may want to see these by flowlet instance, or by run id, so I thought it is useful to keep them here.